### PR TITLE
Add units to UnitConv classes.

### PR DESCRIPTION
**Pull request by T-Nicholls**
**Ready to merge**

### CODE REVIEW

**Comment by T-Nicholls** [comment]
Should we add a get_units(field) method or are we content with the get_unitconv(field).eng/phys_units syntax?

**Comment by T-Nicholls** [comment]
@willrogers I think this should be good to merge now.

File: utils/load_unitconv.m [hunk 1]
```diff
@@ -14,36 +13,66 @@ function load_unitconv(ringmode, renamedIndexes)
...
-fprintf(f_units, 'el_id,field,uc_type,uc_id\n');
+fprintf(f_units, 'el_id,field,uc_type,uc_id,phys_units,eng_units\n');
 fprintf(f_poly, 'uc_id,coeff,val\n');
 fprintf(f_pchip, 'uc_id,eng,phy\n');
 
 quad_families = findmemberof('QUAD');
 sext_families = findmemberof('SEXT');
 
+% Unit conversions for lattice fields
+fprintf(f_units, '%d,%s,poly,%d,%s,%s\n', 0, 'energy', 0, 'Mev', 'Mev');
```
Comment (willrogers) [inline utils/load_unitconv.m]: Should be `MeV` and `GeV` (`GeV` being whichever is the smaller number)

File: utils/load_unitconv.m [hunk 2]
```diff
@@ -59,7 +88,7 @@ function load_unitconv(ringmode, renamedIndexes)
         hcor_index = hcor_index + 1;
     end
     id = write_linear_data(data.field(2) / data.current(2), 0);
-    fprintf(f_units, '%d,%s,poly,%d\n', renamedIndexes(hcor_index), 'x_kick', id); 
+    fprintf(f_units, '%d,%s,poly,%d,%s,%s\n', renamedIndexes(hcor_index), 'x_kick', id, 'rads^-2', 'A');
```
Comment (willrogers) [inline utils/load_unitconv.m]: Not sure where `rads^-2` came from. The value is in radians, but for simplicity we are treating radians as dimensionless: https://en.wikipedia.org/wiki/Radian#Dimensional_analysis

File: utils/load_unitconv.m (hunk 1 above)
Comment (T-Nicholls) [reply utils/load_unitconv.m]: Good catch I will correct the physics units to Gev.

File: utils/load_unitconv.m (hunk 2 above)
Comment (T-Nicholls) [reply utils/load_unitconv.m]: In that case, should it be 'rads', 'A' or '', 'A'?

Comment (willrogers) [reply utils/load_unitconv.m]: '', 'A'

**Pull request merged**